### ISSUE TITLE
Problem installing pyfoam

### ISSUE THREAD

**Issue by nathanfranklin** [simphony/simphony-openfoam#45]
Command with error message:
```
 ~/simphony-framework/src/simphony-openfoam$ ./install_external.sh 
svn: OPTIONS of 'https://svn.code.sf.net/p/openfoam-extend/svn/trunk/Breeder/other/scripting/PyFoam': could not connect to server (https://svn.code.sf.net)
```

**Comment by nathanfranklin** [comment]
The README of simphony-openfoam says *Package foam_controlwrapper requires python 2.7.x, OpenFOAM 2.2.2 and pyFoam 0.6.4* but the command checks out and installs the latest version.

Maybe it would be more appropriate to use https://openfoamwiki.net/images/3/3b/PyFoam-0.6.4.tar.gz (from https://openfoamwiki.net/index.php/Contrib/PyFoam#Downloads).

**Comment by nathanfranklin** [comment]
I would also imagine that the `setup.py` should be updated to reflect this PyFoam required (e.g. `install_requires=...`)

**Comment by nathanfranklin** [comment]
> Maybe it would be more appropriate to use https://openfoamwiki.net/images/3/3b/PyFoam-0.6.4.tar.gz (from https://openfoamwiki.net/index.php/Contrib/PyFoam#Downloads). 

I have a commit so that simphony-framework now uses 0.6.4 (instead of the latest): https://github.com/simphony/simphony-framework/commit/7432496d4380ec7348d7485dc7a33a3c4c41a04a

**Comment by nathanfranklin** [comment]
@norbertonigro @jmarcelogimenez @khiltunen , can anyone pick this issue up?

**Comment by santiagomarquezd** [comment]
Hi @nathanfranklin, pyFoam is required only in I/O wrapper.

**Comment by khiltunen** [comment]
I can make those changes to setup.py and install_external script

**Comment by nathanfranklin** [comment]
Fixed by #47
